### DISCUSSION TITLE
Deploy to a custom domain with a specific path ie: example.com/my-path

### DISCUSSION THREAD

**Discussion by Flow-Fly** [community/community#43716] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello!

I am able to deploy a repo to my custom domain but I would like it to be deployed on a sub-path ie:  `https://example.com/my-path`

I could not find any resources on that, is it something possible?

**Comment by thuongtruong1009** [comment]
@Flow-Fly 

***Step 1:*** In the settings page of a third-party deploying service like Vercel or Github page... You go to the settings section of the domain manager. Find the add domain button. It will show you a list of 2 ranges of records for you to use. Copy each of their fields in turn to proceed to step 2

![image](https://user-images.githubusercontent.com/71834167/211308172-fe68158f-7f9a-4e82-a72c-3bcf1c15bea5.png)

***Step 2:*** Log in to your domain management website. Find where settings manage records. Click the add record button. Fill in the 2 ranges of records that you copied in the previous step in each field.

![image](https://user-images.githubusercontent.com/71834167/211308758-a96e522c-515c-46d6-9a1e-6c537ddd8456.png)

***Finally:*** wait 5 minutes for the server to verify your domain and SSL certificate, then your domain will be really live

* Ở ví dụ trên tôi dùng custom domain trên dịch vụ deploy thứ ba của Vercel. Bạn có thể áp dụng các bước tương tự cho nhưng trang dịch vụ khác...
* Document more at [Vercel-domain](https://vercel.com/docs/concepts/projects/domains#dns-records)

**Reply by Flow-Fly** [reply]
Thanks but this is not my question, I am already able to do that, my website is deployed on `https://example.com` I would like it to be on `https://example.com/my-path`

**Comment by Flow-Fly** [comment, marked as answer]
For anyone interested, found out that is is actually [not yet supported](https://github.com/actions/deploy-pages/issues/73)
